host/CMakeLists.txt

```diff
@@ -146,6 +146,12 @@ if(WIN32)
     target_link_libraries(oehost PUBLIC ws2_32)
 endif()
 
+
+if(OE_SGX)
+  target_link_libraries(oehost PRIVATE oedebugrt)
```

Comment:
Currently host-side oedebugrt is an object library that gets subsumed into liboehost.a via the PRIVATE link libraries definition.

Since oehost is exported, cmake requires oedebugrt to be exported as well eventhough oedeburt does not produce an archive.

This is a temporary state and will change once we support Windows where oedebugrt will be a DLL. Leaving the code/comment as is for now.

Captured action items in #2022